tools/deb/envoy_bootstrap_v2.json

```diff
@@ -39,6 +39,10 @@
       {
         "tag_name": "listener_address",
         "regex": "^listener\\.(((?:[_.[:digit:]]*|[_\\[\\]aAbBcCdDeEfF[:digit:]]*))\\.)"
+      },
+      {
+        "tag_name": "mongo_prefix",
+        "regex": "^mongo\\.(.+?)\\.(collection|cmd|cx_|op_|delays_|decoding_)(.*?)$"
       }
```

Comment:
Can we preemptively add mysql and Kafka please?

Reply:
Do you have a way to test / validate those filters?  I'm happy to add whatever is needed, but I lack a good validation setup.